target/linux/ipq40xx/image/Makefile

```diff
@@ -164,6 +164,15 @@ define Device/avm_fritzrepeater-3000
 endef
 TARGET_DEVICES += avm_fritzrepeater-3000
 
+define Device/cilab_meshpoint-one
+	$(call Device/8dev_jalapeno)
+	DEVICE_DTS := qcom-ipq4018-meshpoint-one
+	DEVICE_VENDOR := Crisis Innovation Lab
+	DEVICE_MODEL := MeshPoint.One
+	DEVICE_PACKAGES := kmod-i2c-gpio kmod-iio-bmp280-i2c kmod-hwmon-ina2xx kmod-rtc-pcf2127
+endef
+TARGET_DEVICES += cilab_meshpoint-one
```

Comment:
As stated earlier, please avoid that and create a common definition for jalapeno board (following the same logic why we have a DTSI and not derive one DTS from the other).